third_party/amd/lib/TritonAMDGPUTransforms/RemoveLayoutConversions.cpp

```diff
@@ -148,51 +148,9 @@ class LayoutPropagation {
 
 } // namespace
 
-// Look ahead to at the transitive uses and see if there is a convert to mma
-// operations.
-static bool hasConvertToMMATransisitiveUse(Operation *op, Attribute encoding) {
```

Comment:
I would leave this pass as is to avoid noise, it really has to be removed/de-duplicated.